### DISCUSSION TITLE
What type of encoding is used for share links?

### DISCUSSION THREAD

**Discussion by gangradem** [AykutSarac/jsoncrack.com#156] [Q&A] [answered]
I want to be able to give a link to JsonCrack directly in my API with all the JSON attached to it:

https://jsoncrack.com/editor?json=[JSONGOESHERE]

When I use the JavaScript [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent), it gives me an error.

**Comment by AykutSarac** [comment, marked as answer]
It uses "compress-json" currently, however I'll be replacing it with shortlinks in a few weeks.
